src/commands/veloce/debug/push.ts

```diff
@@ -50,11 +51,63 @@ export default class Org extends DebugSfdxCommand {
 
     const memberMap = splitMembers(members);
 
+    await this.sendModel(debugSession, rootPath, memberMap['model']);
     await this.sendDrools(debugSession, rootPath, memberMap['drl']);
     // Return an object to be displayed with --json
     return {};
   }
 
+  private findAllModels(sourcepath: string): Set<string> {
+    const result = new Set<string>();
+    const filenames = readdirSync(`${sourcepath}/model`);
+    filenames.forEach((file) => {
+      result.add(file);
+    });
+    return result;
+  }
+
+  private async sendModel(debugSession: DebugSessionInfo, rootPath: string, member: Member) {
+    if (!member) {
+      return;
+    }
+    const params = {
+      veloceNamespace: '',
+      instanceUrl: `${debugSession.instanceUrl}`,
+      organizationId: `${debugSession.orgId}`,
+      oAuthHeaderValue: 'Dummy',
+    };
+    const authorization = Buffer.from(JSON.stringify(params)).toString('base64');
```

Comment:
```
const authorization = getAuthToken({
      veloceNamespace: '',
      instanceUrl: debugSession.instanceUrl,
      organizationId: debugSession.orgId,
      oAuthHeaderValue: 'Dummy',
    });
```

And also, to reduce boilerplate we always can move shared code into `DebugSfdxClass`